synapse/axon.py

```diff
@@ -983,7 +960,8 @@ def fs_readdir(self, path):
         for tufo in tufos:
             fpath = tufo[1].get('axon:path')
             fname = fpath.split('/')[-1]
-            files.append(fname)
+            if fname:
```

Comment:
This is the main change to AxonFS: when reading a dir, check that fname is not empty before appending it to the list of files (due to norm change)